lib/travis/build/script/nix.rb

```diff
@@ -9,7 +9,8 @@ module Build
     class Script
       class Nix < Script
         DEFAULTS = {
-          nix: '2.0.4'
+          nix: '2.0.4',
+          nixpkgs: 'unstable'
```

Comment:
Could we make this a map so that you could:

```
channels:
  nixpkgs: 'https://nixos.org/channels/nixos-unstable'
```

and it runs:

```
sh.cmd "nix-channel --add https://nixos.org/channels/nixpkgs-unstable nixpkgs"
```

?

Reply:
Sure thing.  What do you think about allowing the following as a shorthand:

```rb
channels:
  nixpkgs: '18.09'
```

Which would be equivalent to:

```rb
channels:
  nixpkgs: 'https://nixos.org/channels/nixos-18.09'
```

?

Reply:
I went ahead and added this change in along with the shorthand and a few more tests.  Let me know if you have any feedabck and I'm happy to add more changes.

Reply:
As per @grahamc's comment below, I've gone ahead and removed the shorthand, since it sounds like `nix-channel` has a built-in shorthand that we can use here anyways.